java/bin/start.sh

```diff
@@ -115,3 +155,11 @@ fi
 echo "Triggering Spark Submit job"
 echo ${command} "$@"
 ${command} "$@"
+spark_status=$?
+
+if test $spark_status -eq 0
+then
+        printf "\n Spark command ran fine \n"
```

Comment:
No value in printing this line.

Reply:
Correspondingly made changes in the code.